src/components/ebay-section-title/examples/03-cta/template.marko

```diff
@@ -0,0 +1,5 @@
+<ebay-section-title>
+    <ebay-section-title-title>Today’s Deals – All With Free Shipping</ebay-section-title-title>
+    <ebay-section-title-subtitle>Plus, guaranteed best prices.</ebay-section-title-subtitle>
+    <ebay-section-title-cta href="https://www.ebay.com">See All</ebay-section-title-cta>
```

Comment:
So I would almost say that for CTA, we should just have an option on the `<ebay-section-title>` tag. Something like link and text. If there's no text, it just shows the arrow. If the link option is passed, then it has the `<ebay-section-title-cta>` markup.

Reply:
@agliga I removed `cta` child and added `href` and `cta-text` to the main `ebay-section-tag` tag. Let me know if you any other suggestions for `cta-text` attribute name.